createWorkerTransport.js

```diff
@@ -0,0 +1,14 @@
+const createNodeWorkerTransport = (worker) => {
+  return () => {
+    return {
+      send: (envelope) => {
+        worker.postMessage(envelope);
```

Comment:
My only concern here is that there might be some back-pressure issues here in high volume scenarios, but that should be no problem with the volume we have with jest